Panels/tests/Panels-specs.js

```diff
@@ -1,297 +1,251 @@
+			render(
+				<Panels index={1}>
+					<Panel>
+						<Header />
+					</Panel>
+					<Panel>
+						<Header />
+					</Panel>
 				</Panels>
 			);
 
-			panels.setProps({
-				index: 1
-			});
-
-			const expected = 'default-element';
-			const actual = panels.find('DivPanel#p2').prop('autoFocus');
+			const backButton = screen.getByLabelText('go to previous');
 
-			expect(actual).toBe(expected);
-		}
-	);
+			expect(backButton).toBeInTheDocument();
+		});
 
-	test(
-		'should not set {autoFocus} on child when navigating to a higher index when it has an autoFocus prop set',
-		() => {
-			const DivPanel = ({autoFocus, id}) => <div id={id}>{autoFocus}</div>;
-			const panels = mount(
-				<Panels index={0}>
-					<DivPanel />
-					<DivPanel id="p2" autoFocus="last-focused" />
+		test('should not render back button when not on the first panel and \'noBackButton\' is set to true', () => {
+			render(
+				<Panels index={1} noBackButton>
+					<Panel>
+						<Header />
+					</Panel>
+					<Panel>
+						<Header />
+					</Panel>
 				</Panels>
 			);
 
-			panels.setProps({
-				index: 1
-			});
+			const backButton = screen.queryByLabelText('go to previous');
 
-			const expected = 'last-focused';
-			const actual = panels.find('DivPanel#p2').prop('autoFocus');
+			expect(backButton).toBeNull();
+		});
 
-			expect(actual).toBe(expected);
-		}
-	);
+		test('should not render back button when \'noBackButton\' is set on `Panel` 2', () => {
+			render(
+				<Panels index={1}>
+					<Panel>
+						<Header />
+					</Panel>
+					<Panel noBackButton>
+						<Header />
+					</Panel>
+				</Panels>
+			);
 
-	test(
-		'should return a ref to the root Panel node',
-		() => {
-			const ref = jest.fn();
-			mount(
-				<Panel ref={ref} />
+			const backButton = screen.queryByLabelText('go to previous');
+
+			expect(backButton).toBeNull();
+		});
+
+		test('should render back button on panel 3 when \'noBackButton\' is set on panel 2', () => {
+			render(
+				<Panels index={2}>
+					<Panel>
+						<Header />
+					</Panel>
+					<Panel noBackButton>
+						<Header />
+					</Panel>
+					<Panel>
+						<Header />
+					</Panel>
+				</Panels>
 			);
 
-			const expected = 'ARTICLE';
-			const actual = ref.mock.calls[0][0].nodeName;
+			const backButton = screen.getByLabelText('go to previous');
+
+			expect(backButton).toBeInTheDocument();
+		});
+
+		test('should set back button "aria-label" to backButtonAriaLabel', () => {
+			const label = 'custom back button label';
+			render(
+				<Panels backButtonAriaLabel={label} index={1}>
+					<Panel>
+						<Header />
+					</Panel>
+					<Panel>
+						<Header />
+					</Panel>
+				</Panels>
+			);
 
-			expect(actual).toBe(expected);
-		}
-	);
+			const backButton = screen.getByLabelText(label);
+			const expectedClass = 'slotBefore';
+
+			expect(backButton).toBeInTheDocument();
+			expect(backButton.parentElement.parentElement).toHaveClass(expectedClass);
```

Comment:
You could create a `backButtonContainer` (or so) variable for backButton.parentElement.parentElement. For all the below as well.